app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<p>In this page the requested user (based on the received parameter) will be displayed.</p>
+<div id="users-show">
+  <div class="user-card">
+    <img src="<%= @user.photo %>" alt="User photo">
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p> Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+
+  <div class="user-bio">
+    <h2 class="section-title">Bio</h2>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <ul class="user-posts">
+    <% @posts.each.with_index(1) do |post, index| %>
+      <li class="post">
+        <h2 class="post-title">
+          <%= post.title %>
+          <span class="post-id">#<%= index %></span>
+        </h2>
+        <p class="post-body"><%= post.text %></p>
+        <div class="post-info">
+          <p>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+        </div>
+      </li>
+    <% end %>
+    <li class="user-posts-more">
+      <button>See all posts</button>
+    </li>
```

Comment:
- [ ] This button `See all Post`  opens the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)